app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -0,0 +1,146 @@
+    ppoCards: List<PPOCardDataMock> = listOf(),
+) {
+    Box(
+        modifier = Modifier.fillMaxSize(),
+        contentAlignment = Alignment.Center
+    ) {
+        Scaffold(
+            modifier = modifier,
+            topBar = {
+                TopToolBar(
+                    title = stringResource(id = R.string.project_alerts_fpo),
+                    titleColor = colors.kds_support_700,
+                    leftOnClickAction = onBackPressed,
+                    leftIconColor = colors.kds_support_700,
+                    leftIconModifier = Modifier.testTag(PledgedProjectsOverviewScreenTestTag.BACK_BUTTON.name),
+                    backgroundColor = colors.kds_white,
+                )
+            },
+            backgroundColor = colors.kds_white
+        ) { padding ->
+            LazyColumn(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .fillMaxHeight()
+                    .padding(
+                        start = dimensions.paddingMedium,
+                        end = dimensions.paddingMedium,
+                        top = dimensions.paddingMedium
+                    )
+                    .padding(paddingValues = padding),
+                state = lazyColumnListState
+            ) {
+                item {
+                    Text(
+                        text = stringResource(id = R.string.alerts_fpo, alertsAmount),
+                        style = typography.title3Bold,
+                        color = colors.textPrimary
+                    )
+                }
+
+                items(
+                    items = ppoCards
+                ) { ppoCard ->
+                    Spacer(modifier = Modifier.height(dimensions.paddingMedium))
+
+                    PPOCardView(
+                        viewType = ppoCard.viewType,
+                        onCardClick = { },
+                        projectName = ppoCard.projectName,
+                        pledgeAmount = ppoCard.pledgeAmount,
+                        imageUrl = ppoCard.imageUrl,
+                        imageContentDescription = ppoCard.imageContentDescription,
+                        creatorName = ppoCard.creatorName,
+                        sendAMessageClickAction = { },
+                        shippingAddress = ppoCard.shippingAddress,
+                        showBadge = ppoCard.showBadge,
+                        onActionButtonClicked = { },
+                        onSecondaryActionButtonClicked = { },
+                        timeNumberForAction = ppoCard.timeNumberForAction
+                    )
+                }
+
+                item {
+                    Spacer(modifier = Modifier.height(dimensions.paddingDoubleLarge))
+                }
+            }
+        }
+    }
+}
+
+enum class PledgedProjectsOverviewScreenTestTag {
+    BACK_BUTTON,
+}
+
+// For preview purposes only, will remove once we have the PPO Card payload model from graph
+data class PPOCardDataMock(
```

Comment:
👍  This can be the model used once the query is ready, same as we use `CreatePaymentIntentInput` data class to not speak directly with data types defined on GraphQL. I'm assume the query will return the same information you are displaying here except for the Click Actions